CUDADataFormats/TrackingRecHit/src/TrackingRecHit2DCUDA.cc

```diff
@@ -0,0 +1,19 @@
+#include "CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DCUDA.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/copyAsync.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
+
+template <>
+cms::cuda::host::unique_ptr<float[]> TrackingRecHit2DCUDA::localCoordToHostAsync(cudaStream_t stream) const {
+  auto ret = cms::cuda::make_host_unique<float[]>(4 * nHits(), stream);
+  cms::cuda::copyAsync(ret, m_store32, 4 * nHits(), stream);
+  return ret;
+}
+
+template <>
+cms::cuda::host::unique_ptr<uint32_t[]> TrackingRecHit2DCUDA::hitsModuleStartToHostAsync(cudaStream_t stream) const {
+  auto ret = cms::cuda::make_host_unique<uint32_t[]>(2001, stream);
```

Comment:
I assume you are probably right, but I would prefer @VinInn to confirm; since there are likely other instances, Ive added this to [#32483 (comment)](https://github.com/cms-sw/cmssw/issues/32483#issuecomment-744747495)

Reply:
left over from early times.

Reply:
done in cms-patatrack#593 / cms-patatrack@f6924bb